source/production/F0.Minesweeper.Logic/Minelayer/FirstEmptyMinelayer.cs

```diff
@@ -14,38 +14,12 @@ IEnumerable<Location> IMinelayer.PlaceMines(IEnumerable<Location> possibleLocati
 
 		private static IEnumerable<Location> RemoveLocationsAroundClickedLocation(IList<Location> locations, Location clickLocation)
```

Comment:
I'd rename this to `RemoveClickedLocationArea` since we not only remove the locations around the clicked cell but the location of the clicked cell as well.

Reply:
Nice, that sounds better 😄